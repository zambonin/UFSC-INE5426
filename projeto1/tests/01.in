  int a_
int	BB, c
int d=   0,	e1 =1

a_ = d+2* 3
BB = (-a_)	/  12-1
	c = e1*e1/a_


int Abra = 1, Kadabra, Alakazam

Kadabra = Abra * 10 + 1
Alakazam = 1 + Kadabra * 10

int x10_12Y
x10_12Y = - (10 + 2) * (3 - 4) / (-0 + 1)

