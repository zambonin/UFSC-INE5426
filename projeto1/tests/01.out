int var: a_
int var: BB, c
int var: d = 0, e1 = 1
= a_ + d * 2 3
= BB - / -u a_ 12 1
= c / * e1 e1 a_
int var: Abra = 1, Kadabra, Alakazam
= Kadabra + * Abra 10 1
= Alakazam + 1 * Kadabra 10
int var: x10_12Y
= x10_12Y / * -u + 10 2 - 3 4 + -u 0 1
